astroid/node_classes.py

```diff
@@ -1051,6 +1051,19 @@ def postinit(self, targets=None, value=None):
         self.value = value
 
 
+class AnnAssign(mixins.AssignTypeMixin, Statement):
+    """class representing an AnnAssign node"""
+    _astroid_fields = ('targets', 'value', 'annotation',)
+    targets = None
```

Comment:
this likely should be named `target`, not `targets`. Annotated assign cannot be done anyway:

```
>>> (a, b): (int, int)
  File "<stdin>", line 1
SyntaxError: only single target (not tuple) can be annotated
```

Also, `help(_ast.AnnAssign)` yields:

    _fields = ('target', 'annotation', 'value', 'simple')

Astroid is supposed to be superset of `ast` and should be consistent with it. I'll recommend changing `_astroid_fields` value quoted above.

[PEP](https://www.python.org/dev/peps/pep-0526/#runtime-effects-of-type-annotations) also mentions _simple name_, but I don't think it affects us in any way.